app/routines/update_user_salesforce_lead_info.rb

```diff
@@ -0,0 +1,24 @@
+class UpdateUserSalesforceLeadInfo
+  def self.call
+    new.call
+  end
+
+  def call
+    leads ||= OpenStax::Salesforce::Remote::Lead.where('status != ?','Converted').select(:id, :grant_tutor_access).to_a
```

Comment:
![Screen Shot 2021-10-13 at 1 10 47 PM](https://user-images.githubusercontent.com/3905516/137189361-2ed92fef-9522-4d86-a931-90aaef03f338.png)

Just so you are aware, I ran this query on production and it returns over 500k rows. So this will probably be VERY slow and memory-consuming.